defaults/main.yml

```diff
@@ -161,6 +164,41 @@ docker__listen: [ '{{ docker__tcp_listen }}' ]
 docker__labels: {}
 
                                                                    # ]]]
+# .. envvar:: docker__debug [[[
+#
+# Start docker daemon in debug mode.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__debug: true
+#
+docker__debug: false
+
+                                                                   # ]]]
+# .. envvar:: docker__live_restore [[[
+#
+# Enables keeping containers alive during daemon downtime.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__live_restore: true
+#
+docker__live_restore: []
+                                                                   # ]]]
+# .. envvar:: docker__graph [[[
+#
+# Alternative root path of the docker runtime.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__graph: /data/docker
+#
+
+docker__graph: []
```

Comment:
Is there a default `docker__graph` path? if yes, it should be specified in the defaults, unless there are issues with that?